backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -46,14 +46,15 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       
       Camera camera = getCameraController().getCameraById(Integer.parseInt(camID));
       String tags = request.getParameter("tags");
+      String name = request.getParameter("name");
```

Comment:
Maybe you could test here if the name is not null or do we allow a preset without a name?